### ISSUE TITLE
Show recent task sequences on project level

### ISSUE THREAD

**Issue by johannes-b** [keptn/keptn#2506]
This feature depends on: #3536 

With the started/finished events of task sequences - as proposed [here](https://github.com/keptn/enhancement-proposals/pull/6/files#diff-849e5499f3f38664566e5d9fbe0755c9R99) - this information can be displayed in the project overview for each project.

**Task:**
- Show the last 5 task sequences that were triggered for this project in the project tile
- One sequence is identified by: `icon`, `<service> in <stage>`, `sequence name + started/finished`, `x min/sec/hours/days ago` the sequence was triggered.
  - Table with four columns: icon | service/stage | sequence | time
  - max five rows
  - ordered by the last column
- The icon: 
  - when the sequence is started but not finished -> show a loading icon
  - depending on the result, turn the icon color into red/green
  - *be opinionated*: support the icons for deployment (small box), evaluation (traffic light), remediation (escalation mark) 

- Build it as a re-usable component to be used in the environment screen (even though the environment screen is out of scope of this issue)

**Definition of Done:**

![image](https://user-images.githubusercontent.com/729071/95575454-06dfc100-0a2f-11eb-9a33-ebfd2ad831e2.png)

**Comment by Kirdock** [comment]
Currently we can't provide a "good" solution for this issue without touching the backend. In order to get the result `finished`, `triggered` etc. for each sequence we have to load all its traces.
Let's say we have 10 projects with 3 stages each. 10 x 5 sequences are loaded and then let's say with an average of 6 tasks (6 x 3 Traces) per stage we would load `10 x 5 x 3 x 6 x 3 = 2,700` traces and we would have `1 (get projects) + 10 (recent 5 sequences per project) + 10 x 5 (traces for each sequence) = 61` API-Calls.
This is going to lead to performance issues.

One possible solution would be, to save the result of a sequence in the sequence (e.g. a number determines `loading`, `finished successfully`, `finished failed` or `waiting for approval`). Currently the result is determined when we iterate through all traces of a sequence.
